frontend/ui/data/keyboardlayouts/zh_CN_keyboard.lua

```diff
@@ -0,0 +1,155 @@
+local IME = require("frontend/ui/data/keyboardlayouts/generic_ime")
+local util = require("util")
+local _ = require("gettext")
+
+-- Start with the english keyboard layout (deep copy, to not alter it)
+local py_keyboard = require("util").tableDeepCopy(require("ui/data/keyboardlayouts/en_keyboard"))
+local SHOW_CANDI_KEY = "keyboard_chinese_pinyin_show_candidates"
```

Comment:
SETTING_NAME instead of KEY - to make it more obvious what it is.

Reply:
Made a table that defaults to`{show_candi=true} ` for pinyin settings. As there might be other possible ones in the future (candi separator, use numbers to choose candidates, etc)